templates/e-commerce/data/build.js

```diff
@@ -0,0 +1,107 @@
+const fs = require('fs');
```

Comment:
This is a simple node JS script that generates some products for us to use on the listings page. It creates all endpoints for 4 categories with two filters (price high to low and vice versa). 

Not sure if this file belongs here or just the output files. What do you think?